src/AutoMapper/ConfigurationStore.cs

```diff
@@ -394,7 +405,8 @@ public TypeMap ResolveTypeMap(TypePair typePair)
                                 FindTypeMapFor(tp) ??
                                 (!CoveredByObjectMap(typePair)
                                     ? FindConventionTypeMapFor(tp) ??
-                                      FindClosedGenericTypeMapFor(tp)
+                                      FindClosedGenericTypeMapFor(tp) ??
+                                      (CreateMissingTypeMaps ? CreateTypeMap(tp) : null)

```

Comment:
What purpose is CreateMissingTypeMaps for?  Isn't that what conventions are suppose to handle?